<ui version="4.0" >
 <class>Programme</class>
 <widget class="QDialog" name="Programme" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Programme</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="nomProgramme" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Programme</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="chaine" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Début:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Fin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QDateEdit" name="dateDebut" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateFin" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QTimeEdit" name="heureDebut" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="heureFin" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QCheckBox" name="checkBoxAvant" >
              <property name="text" >
               <string>commencer</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAjouter" >
              <property name="text" >
               <string>finir</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLineEdit" name="avant" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ajouter" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>minutes avant</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>minutes après</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Format nom fichier:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nomFichier" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Fichier créé:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="apercuNomFichier" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%n&lt;/span> : Nom de la chaine  &lt;span style=" font-weight:600;">%t&lt;/span> : Titre du programme  &lt;span style=" font-weight:600;">%a&lt;/span> : Année &lt;span style=" font-weight:600;">%m&lt;/span> : Mois  &lt;span style=" font-weight:600;">%j&lt;/span> : Jour&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="desc" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boutonRegarder" >
       <property name="text" >
        <string>&amp;Regarder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonAjouter" >
       <property name="text" >
        <string>&amp;Programmer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fermer" >
       <property name="text" >
        <string>&amp;Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="labelInfo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fermer</sender>
   <signal>clicked()</signal>
   <receiver>Programme</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>617</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel" >
     <x>342</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonAjouter</sender>
   <signal>clicked()</signal>
   <receiver>Programme</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>480</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel" >
     <x>306</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
